src/common/jsonb/src/functions.rs

```diff
@@ -836,6 +836,30 @@ pub fn parse_json_path(path: &[u8]) -> Result<Vec<JsonPath>, Error> {
                     return Err(Error::InvalidToken);
                 }
             }
+        } else if c == b'"' {
+            prev_idx = idx;
+            loop {
+                let c = read_char(path, &mut idx)?;
+                if c == b'\\' {
+                    idx += 1;
+                    let c = read_char(path, &mut idx)?;
+                    if c == b'"' {
+                        idx += 1;
+                    }
+                } else if c != b'"' {
+                    idx += 1;
+                } else {
+                    // Try to read to check if has another '"ab"', if has return err
+                    let c = read_char(path, &mut idx);
+                    match c {
+                        Ok(_) => return Err(Error::InvalidToken),
+                        Err(_) => break,
+                    }
+                }
+            }
+            let s = std::str::from_utf8(&path[prev_idx..idx - 1])?;
+            let json_path = JsonPath::String(Cow::Borrowed(s));
+            json_paths.push(json_path);
```

Comment:
A string value must only have one, need to check the length of `json_paths`, to avoid other values existing before this value.